applications/migrations/0008_applicationstatus.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.0.10 on 2020-12-18 08:57
```

Comment:
Does django freak out now that i merged my pull request and this has same 0008 number as existing migration.

Reply:
Yes kinda, but I'll revert and recreate migrations for this